### ISSUE TITLE
Steam main window GUI is unresponsive unless tabbing to friend GUI

### ISSUE THREAD

**Issue by sanecito** [flathub/com.valvesoftware.Steam#1045]
**Game information**

N/A

**Distribution name and version where applicable**

Fedora Kinoite 37.20230209.0 (this has been happening for several months though; reporting now as no one has seemed to report)

**Flatpak info**

Flatpak 1.14.1

nvidia-525-85-05
default
host

**Problem description**

Pretty consistently for some indeterminate time after starting Steam, the Steam main window GUI is not responsive. Hover over effects for start game buttons, header menu selection don't happen. Similarly, clicking on them doesn't result in any visible action.

However, when switching main focus to the pop-out friend GUI via alt-tab, and mousing over main window GUI elements hover over effects will happen. After tabbing back to the main window GUI, mouse will snap to lower border of the main window and start the click-and-drag/resize until there's a mouse click to exit out of the resizing.

In short, there seems to be something off with where the Steam client thinks there should be cursor focus as part of startup and carrying through until some unknown amount of time/trigger happens to align with expected behavior.

Hidden issue help text says to provide logs, but does not provide how to unlike requests to run `flatpak --gl-drivers`, etc

**Does this issue reproduce with native Steam**

Have not tested

**Comment by nstana** [comment]
I'm getting this problem too on Arch Linux.

**Comment by jntesteves** [comment]
The solution for now is to close the **Friends List** window. Until Valve fixes it, I think.

Upstream issue: ValveSoftware/steam-for-linux#8890

**Comment by sanecito** [comment]
Closing Friends List didn't fix it for me. While nstana reported seeing this with AMD graphics suggesting a Steam based issue and not a GPU based one, this went away for me when I updated my Nvidia GPU to an Intel Arc 750 (but I kept the proprietary Nvidia drivers installed). Steam GUI has consistently been performing as intended for the month since the hardware swap.

For those tracking and seeing this issue, Valve did put out a substantial Steam client beta release which might fix this issue: https://www.gamingonlinux.com/2023/04/huge-steam-beta-upgrade-with-new-overlay-screenshot-manager-more/

So if anyone that is seeing issues could try the beta channel and report back whether it happened to fix things or not, that could be useful for other people seeing the issue.

**Comment by nstana** [comment]
Have yet to experience this again since updating to the latest beta.
